stable/fluent-bit/templates/daemonset.yaml

```
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1beta2
```

It should be `apps/v1`.

DaemonSet, Deployment, and ReplicaSet resources will no longer be served from extensions/v1beta1, apps/v1beta1, or apps/v1beta2 in v1.16. Migrate to the apps/v1 API, available since v1.9. Existing persisted data can be retrieved via the apps/v1 API.

```suggestion
apiVersion: apps/v1
```

---

Got it, thank you

---

You need to sign the DCO - https://github.com/helm/charts/pull/11900/checks?check_run_id=95318557.
You need also to bump the chart version - https://circleci.com/gh/helm/charts/66339?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link.
Please also squash to single commit.